pages/linux/rdesktop.md

```diff
@@ -0,0 +1,28 @@
+# RDESKTOP
```

Comment:
OK, I will change it. But I follow the pages of SSH, which also capitalises the first line `#SSH`.